client/asset/interface.go

```diff
@@ -54,13 +74,15 @@ type Wallet interface {
 	FindRedemption(ctx context.Context, coinID dex.Bytes) (dex.Bytes, error)
 	// Refund refunds a contract. This can only be used after the time lock has
 	// expired.
-	Refund(Receipt) error
+	Refund(Receipt, *dex.Asset) error
 	// Address returns an address for the exchange wallet.
 	Address() (string, error)
 	// Unlock unlocks the exchange wallet.
 	Unlock(pw string, dur time.Duration) error
 	// Lock locks the exchange wallet.
 	Lock() error
+	// PayFee pays a registration fee to the DEX.
+	PayFee(fee uint64, address string, nfo *dex.Asset) (Coin, error)
```

Comment:
Added this method to both DCR and BTC wallets, with an assumption that the registration fee asset will eventually be a DEX variable.